widget/embedded/src/containers/QuoteInfo/QuoteInfo.tsx

```diff
@@ -55,25 +305,43 @@ export function QuoteInfo(props: PropTypes) {
 
   return showQuote ? (
     <QuoteContainer onClick={() => onClick?.(quote)}>
-      <Quote
-        quote={quote}
-        error={error}
-        showModalFee={showModalFee}
-        warning={warning}
-        tagHidden={tagHidden}
-        selected={selected}
-        type={type}
-        expanded={expanded}
-        onClickAllRoutes={onClickAllRoutes}
-        input={{
-          value: inputAmount,
-          usdValue: inputUsdValue?.toString() ?? '',
-        }}
-        output={{
-          value: outputAmount?.toString() ?? '',
-          usdValue: outputUsdValue?.toString() ?? '',
-        }}
-      />
+      {config?.variant === 'full-expanded' ? (
+        <ExpandedRoute
+          time={time}
```

Comment:
I think you should pass `tooltipContainer` as well.

Reply:
Fixed